dashboard/test/ui/step_definitions/pd.rb

```diff
@@ -51,62 +69,67 @@
+    end: DateTime.new(2016, 3, 15) + 9.hours
+  )
 
   number.to_i.times do |x|
     enrollment = Pd::Enrollment.create(
       first_name: "First name - #{SecureRandom.hex}",
       last_name: "Last name - #{SecureRandom.hex}",
       email: "enrolled_teacher#{x}@foo.com",
       school_info: SchoolInfo.find_or_create_by({
-        country: 'US',
-        school_type: 'other',
-        state: 'WA',
-        zip: '98101',
-        school_name: 'Code.org'
-      }),
+                                                    country: 'US',
+                                                    school_type: 'other',
+                                                    state: 'WA',
+                                                    zip: '98101',
+                                                    school_name: 'Code.org'
+                                                }),
       pd_workshop_id: workshop.id
     )
     PEGASUS_DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: enrollment.id).delete
   end
-  workshop.update(started_at: Time.now)
-  workshop.update(ended_at: Time.now)
 
-  responses = {}
+  if state == 'end'
+    workshop.update(started_at: DateTime.new(2016, 3, 15))
+    workshop.update(ended_at: DateTime.new(2016, 3, 15))
 
-  [
-    Api::V1::Pd::WorkshopScoreSummarizer::FACILITATOR_EFFECTIVENESS_QUESTIONS,
-    Api::V1::Pd::WorkshopScoreSummarizer::TEACHER_ENGAGEMENT_QUESTIONS,
-  ].flatten.each do |question|
-    responses[question] = PdWorkshopSurvey::OPTIONS[question].last
-  end
+    responses = {}
 
-  Api::V1::Pd::WorkshopScoreSummarizer::OVERALL_SUCCESS_QUESTIONS.each do |question|
-    responses[question] = PdWorkshopSurvey::AGREE_SCALE_OPTIONS.last
-  end
+    [
+        Api::V1::Pd::WorkshopScoreSummarizer::FACILITATOR_EFFECTIVENESS_QUESTIONS,
+        Api::V1::Pd::WorkshopScoreSummarizer::TEACHER_ENGAGEMENT_QUESTIONS,
+    ].flatten.each do |question|
+      responses[question] = PdWorkshopSurvey::OPTIONS[question].last
+    end
 
-  workshop.enrollments.each do |enrollment|
-    puts "Enrollment ID - #{enrollment.id}"
-    PEGASUS_DB[:forms].insert(
-      secret: SecureRandom.hex,
-      source_id: enrollment.id,
-      kind: 'PdWorkshopSurvey',
-      email: enrollment.email,
-      data: responses.to_json,
-      created_at: Time.now,
-      created_ip: '',
-      updated_at: Time.now,
-      updated_ip: ''
-    )
+    Api::V1::Pd::WorkshopScoreSummarizer::OVERALL_SUCCESS_QUESTIONS.each do |question|
+      responses[question] = PdWorkshopSurvey::AGREE_SCALE_OPTIONS.last
+    end
+
+    workshop.enrollments.each do |enrollment|
+      puts "Enrollment ID - #{enrollment.id}"
+      PEGASUS_DB[:forms].insert(
+        secret: SecureRandom.hex,
+        source_id: enrollment.id,
+        kind: 'PdWorkshopSurvey',
+        email: enrollment.email,
+        data: responses.to_json,
+        created_at: Time.now,
+        created_ip: '',
+        updated_at: Time.now,
+        updated_ip: ''
+      )
+    end
```

Comment:
We don't need survey responses to test the general workshop dashboard UI. This could be a separate step "And I complete surveys", or an option on this step.